platform/error/platform-error-impl/package.json

```diff
@@ -44,7 +38,7 @@
     "node-fs": "0.1.7",
     "node-options": "0.0.3",
     "path": "0.4.9",
-    "casperjs": "~1.1.0",
+    "casperjs": "^1.1.0-beta3",
```

Comment:
Phil: while this particular case it's an arguably small risk, I do agree with you. But this is not the only dependency using semver ranges.
Question then (for all interested) becomes: Should I:
a. just remove the caret from the casperjs line on the three affected package.jsons?
b. investigate (pretty easy) which particular versions the other dependencies in the three "casperjs" package.jsons resolve to (for the ones using ranges) and set those "statically" as well?
c. Do "b." but for all 9 package.json files that I'm touching with this PR?
c.